apps/shipments/pagination.py

```diff
@@ -0,0 +1,22 @@
+"""
+Copyright 2019 ShipChain, Inc.
+ Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from rest_framework_json_api.pagination import PageNumberPagination
+
+
+class ListDataPagination(PageNumberPagination):
+
+    def get_paginated_response(self, data):
+        response = super(ListDataPagination, self).get_paginated_response(data)
+        response.data['data'] = response.data.pop('results')
```

Comment:
This seems to be something we're going to need to do frequently when adding pagination to custom views. Can you please rename this class to something generic in both this branch and shipment history, so we can reuse this code?

Reply:
I do agree, for now I named this class: `ListDataPagination` in the optic to change / rename the shipment history this same way or keep it in the rebasing phase from the first merged branch.
Do you have some other name in mind? What about `GenericListDataPagination`?

Reply:
How about `CustomResponsePagination`? I think it would make sense to move it to apps/pagination.py too since it's generic :+1:

Reply:
Woks for me.